src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+        else task = new Deadline(arr[2]);
+        if(arr[1].trim().charAt(0) == '1') task.isCompleted = true;
+        else task.isCompleted = false;
+        return task;
+    }
+    private static String convertToFileFormat(Task task) {
+        String fileString = "";
+        if(task instanceof Todos) fileString += "T | ";
+        else if(task instanceof Deadline) fileString += "D | ";
+        else if(task instanceof Event) fileString += "E | ";
+        if(task.isCompleted) fileString += "1 | ";
+        else fileString += "0 | ";
+        fileString += task.description;
+        return fileString;
+    }
+    private static void load() throws IOException, EmptyArgumentException {
+        try {
+            File file = new File(filePath);
+            Scanner sc = new Scanner(file);
+            while(sc.hasNext()) {
+                String line = sc.nextLine();
+                Task task = convertToTaskFormat(line);
+                tasks.add(task);
+                System.out.println("hi");
+            }
+        }
+        catch(FileNotFoundException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+
+
+    private static void addItem(String typeOfEvent, String eventDescription) throws EmptyArgumentException, InvalidCommandException {
+        if(typeOfEvent.equals("todo")) {
+            tasks.add(new Todos(eventDescription));
+        }
+        else {
+            String[] _eventDescription = eventDescription.split("/");
+            StringBuilder firstPart = new StringBuilder(_eventDescription[0]).append(" ");
+            StringBuilder secondPart = new StringBuilder(_eventDescription[1]);
+            secondPart.insert(0, '(');
+            secondPart.insert(3, ':');
+            secondPart.append(')');
+            firstPart.append(secondPart);
+            if (typeOfEvent.equals("deadline")) tasks.add(new Deadline(firstPart.toString()));
+            else if (typeOfEvent.equals("event")) tasks.add(new Event(firstPart.toString()));
+        }
+        System.out.print("added: ");
+        System.out.println(tasks.get(tasks.size()-1));
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void deleteItem(int itemNo) {
+        Task toBeDeleted = tasks.get(itemNo-1);
+        tasks.remove(itemNo-1);
+        System.out.println("Noted.I have removed this task:");
+        System.out.print("  ");
+        System.out.println(toBeDeleted);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void markItemAsDone(int itemNo) {
+        tasks.get(itemNo-1).isCompleted = true;
+        System.out.println("Nice, I have marked this task as done!");
+        System.out.print("  ");
+        System.out.print(tasks.get(itemNo-1));
+    }
+
+    private static void listItems() {
+        System.out.println("here are your tasks:");
+        for(int itemNo=1;itemNo<=tasks.size();itemNo++) {
```

Comment:
should there be a space after for ?
and should operators be surrounded by a space character?